### PR TITLE
Adding HTAN buckets

### PR DESCRIPTION
**Pull request by aditigopalan**
Creating projects for htan2

### CODE REVIEW

**Review by adamjtaylor — APPROVED** [review]

File: config/prod/htan2-crc-syn63298073.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.8.3/templates/S3/synapse-external-bucket.j2"
+stack_name: "htan2-crc-syn63298073"
+stack_tags:
+  OwnerEmail: 'aditi.gopalan@sagebase.org'
+  CostCenter: "HTAN-DFCI / 120100"
```
Comment (zaro0508) [inline config/prod/htan2-crc-syn63298073.yaml:8]: should be `DFCI HTAN / 120100` to match code at https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2553544733/Finance+Program+Codes+for+AWS+tagging

**Review by zaro0508 — CHANGES_REQUESTED** [review]
Do we still need [htan2-testing1 bucket](https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/config/prod/htan2-testing1.yaml)?

**Comment by adamjtaylor** [comment]
> Do we still need [htan2-testing1 bucket](https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/config/prod/htan2-testing1.yaml)?

Yes please @zaro0508  - we have this backing a Synapse project used for further and ongoing testing

File: config/prod/htan2-crc-syn63298073.yaml (hunk 1 above)
Comment (aditigopalan) [reply config/prod/htan2-crc-syn63298073.yaml]: changed, thank you!

**Review by adamjtaylor — APPROVED** [review]

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**